source/include/BuildConfig.h

```diff
@@ -233,6 +233,7 @@ SDG
 #endif
 
 // TODO(PaulAustin): allow for thread locals on linux/unix
+#undef VIVM_THREAD_LOCAL
```

Comment:
Try the following instead:
#ifndef VIVM_THREAD_LOCAL
#define VIVM_THREAD_LOCAL
#endif